src/pages/project/project_wizard/RelatedRegistrationsField.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material';
+import { useFormikContext } from 'formik';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFetchRegistrationsForProject } from '../../../api/hooks/useFetchRegistrationsForProject';
+import { ListPagination } from '../../../components/ListPagination';
+import { ListSkeleton } from '../../../components/ListSkeleton';
+import { RegistrationListItemContent } from '../../../components/RegistrationList';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { CristinProject } from '../../../types/project.types';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
+
+export const RelatedRegistrationsField = () => {
+  const { t } = useTranslation();
+  const { values } = useFormikContext<CristinProject>();
+  const [page, setPage] = useState(1);
+  const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+
+  const resultsQuery = useFetchRegistrationsForProject(values.id, rowsPerPage, page);
+  const results = resultsQuery.data;
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+      <Typography variant="h2">{t('project.form.related_registrations')}</Typography>
+      {values.id ? (
+        <>
+          <Typography>{t('project.form.related_registrations_description')}</Typography>
+          {resultsQuery.isPending ? (
+            <Box>
+              <ListSkeleton arrayLength={4} maxWidth={60} height={20} />
+            </Box>
```

Comment:
Kan bruke native-elementer når vi ikke skal bruke noe av funksjonaliteten MUI baker inn
```suggestion
            <div>
              <ListSkeleton arrayLength={4} maxWidth={60} height={20} />
            </div>
```

Reply:
Det er sant. Akkurat her trengs nok ikke noe.